style/style.css

```diff
@@ -320,3 +333,210 @@ textarea {
 .btn-contact {
   width: 48%;
 }
+
+/*
+      ==============================================
+       *** MEDIA QUERY - Desktop Version- 1024px ***
+      ==============================================
+*/
+
+@media only screen and (min-width: 1024px) {
+  /* ------- Main ------- */
+  main {
```

Comment:
- [ ] If you want that the main section is the only thing that the user sees unless they scroll down, try to use: `height: 100vh` instead of top padding.

Reply:
Solved. Thank you for this hint :)